lntemp/harness_assertion.go

```diff
@@ -345,12 +345,15 @@ func (h *HarnessTest) AssertChannelExists(hn *node.HarnessNode,
 // findChannel tries to find a target channel in the node using the given
 // channel point.
 func (h *HarnessTest) findChannel(hn *node.HarnessNode,
-	chanPoint *lnrpc.ChannelPoint) (*lnrpc.Channel, error) {
+	chanPoint *lnrpc.ChannelPoint, peerAliasLookup bool) (*lnrpc.Channel,
```

Comment:
We could use a variadic argument list with functional options to avoid needing to add the new parameter everywhere. And then just use `withPeerLookup()` if you want to enable it. Example, in case it's not clear what I mean: https://github.com/lightninglabs/lndclient/blob/master/lightning_client.go#L2786

Reply:
Thanks for the hint, this looks a little cleaner now.